out/git_lock_handler.go

```diff
@@ -292,9 +292,10 @@ func (glh *GitLockHandler) messagePrefix() string {
 	buildName := os.Getenv("BUILD_NAME")
 	jobName := os.Getenv("BUILD_JOB_NAME")
 	pipelineName := os.Getenv("BUILD_PIPELINE_NAME")
+	teamName := os.Getenv("BUILD_TEAM_NAME")
```

Comment:
Ah, this comes from the ATC metadata, right ?

Reply:
Yes.